mysql/resource_grant_test.go

```diff
@@ -3,14 +3,15 @@ package mysql
 import (
```

Comment:
Could you please create a test to catch this case / issue?

Reply:
I've fixed the existing ones. Anything else you think should be added?

Reply:
Generally when you want to fix a regression, there should be a new test catching the very issue you are trying to fix.

So that means you should test importing (I know we don't test it anywhere, but still). And ideally, you should leave all previous tests in place. Or fix them, if there is any error there - but there should be some rationale when removing tests.

Reply:
I don't think it was a regression in the sense that a feature existed and it stopped working. It was, as I understand, a new issue that was implemented that never really worked. I think the current test is reasonable, ti was just built in a way that worked with the implementation, but that was not working in the first place. I think the current test covers the implementation and I'm not sure what sort of regression, specifically, we'd need to cover.

Reply:
In the bug https://github.com/petoju/terraform-provider-mysql/issues/112 , you described
```
terraform import mysql_grant.foo_grant_local 'foo_user@localhost@test_db@*'
```
stopped working. Cannot you test import exactly like that here?

I'd do it based on
https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/import#resource-acceptance-testing-implementation

I'm not completely sure how it works - so if that's too difficult and once I find some time, I can also try looking at that.